app/views/layouts/application.html.erb

```diff
@@ -7,7 +7,19 @@
   <%= csrf_meta_tags %>
   <meta name="viewport" content="width=device-width, initial-scale=1"></head>
 <body>
+  <nav>
+    <% if user_signed_in? %>
+      Welcome, <%= current_user.email %>. Not you?
+      <%= link_to "Log out", destroy_user_session_path, method: 'delete' %>
+    <% else %>
+      <%= link_to "Login", new_user_session_path %>
+      <br><p>OR</p><br>
+      <%= link_to "Sign in with Google", user_google_oauth2_omniauth_authorize_path %>
+    <% end %>
+  </nav>
+
```

Comment:
Lines 10-20 were moved from the default layout into the navbar partial. You should move line 17 into home/index.html.erb